src/pages/video-detail-page/video-detail-page.js

```diff
@@ -9,63 +9,55 @@ import {
 import { fetchRelatedVideos } from '../../api';
 import { YOUTUBE_EMBED_URL } from '../../utils';
 
-class VideoDetailPage extends Component {
-  state = {
-    relatedVideos: { items: [], error: false },
-  };
+export const VideoDetailPage = props => {
```

Comment:
// destructure props 
```
export const VideoDetailPage = ({ match}) => {
...
}
```


// follow this link for proptypes definition
https://reactjs.org/docs/typechecking-with-proptypes.html